README.md

```diff
@@ -6,7 +6,7 @@ GOV.UK stack.
 ## How to use
 
 ```shell
-rails new APP_NAME -m http://raw.github.com/govuk-rails-app-template/master/template.rb
+rails new APP_NAME --skip-test-unit --skip-spring -m govuk-rails-app-template/template.rb
```

Comment:
This command adds several gems that we may not want, such as turbolinks and coffee-rails.  We should probably suggest using `--no-javascript` here to omit these.  That will leave out jquery as well, but most apps that use javascript, will be either using govuk_admin_template, or slimmer to provide jquery.

Another option would be to replace the Gemfile with a fully curated one.